docs/example-narratives/examples/seasonal-ratings-proposals-patch.json

```diff
@@ -1,10 +1,52 @@
 {
+  "proposal-header": {
```

Comment:
As we think about this here- do we need a separate set of examples for seasonal ratings for day vs night?

Reply:
Yes, absolutely, thanks for reminding me.